grammars/erlang.bnf

```diff
@@ -321,6 +324,32 @@ argument_definition_list ::= '(' argument_definition? (',' argument_definition)*
 clause_guard ::= when <<guarded guard>> {pin=1}
 clause_body ::= '->' exprs {pin=1}
 
+app_expression ::= empty '{' 'application' ',' module_ref ',' app_parameters '}' {elementType=tuple_expression}
+app_parameters ::= '[' app_parameter? (',' app_parameter)* ']' {elementType=list_expression}
+app_parameter ::= '{'
+   ('description' ',' string_literal
+  | 'id' ',' string_literal
+  | 'vsn' ',' string_literal
+  | 'modules' ',' app_modules
+  | 'maxP' ',' integer
+  | 'maxT' ',' integer
+  | 'registered' ',' app_modules
+  | 'included_applications' ',' app_applications
+  | 'applications' ',' app_applications
+  | 'env' ',' app_env
+  | 'mod' ',' app_mod
+  | 'start_phases' ',' expression
+  | 'runtime_dependencies' ',' app_runtime_deps
+  | config_exprs)
+  '}' {methods=[getName]}
```

Comment:
It seems like a not very good solution. Why have you decided to make changes to ```org.intellij.erlang.rebar.importWizard.ImportedOtpApp``` at all?